src/reanimated2/index.ts

```diff
@@ -91,12 +91,10 @@ export type {
   EasingFactoryFn,
 } from './Easing';
 export { Easing } from './Easing';
-export {
-  measure,
-  dispatchCommand,
-  scrollTo,
-  setGestureState,
-} from './NativeMethods';
+export { measure } from './nativeMethods/measure';
+export { dispatchCommand } from './nativeMethods/dispatchCommand';
+export { scrollTo } from './nativeMethods/scrollTo';
+export { setGestureState } from './nativeMethods/setGestureState';
```

Comment:
We could also re-export those from `NativeMethods.ts` and import everything from `./NativeMethods` here. Also, these are not methods so we could rename it to `NativeFunctions.ts` or even simply `functions.ts`?

Reply:
I renamed `nativeMethods` directory to `functions` and created an `index.ts` file there.